_data/site.yaml

```
@@ -65,16 +65,16 @@ primary_navigation:
 
 catalog_filters:
   - name: Most Viewed Datasets
-    url: https://catalog.data.gov
+    url: https://catalog.data.gov/dataset?q=&sort=views_recent+desc
     iconName: most-viewed-datasets-icon
   - name: Recently Added Datasets
-    url: https://catalog.data.gov
+    url: https://catalog.data.gov/dataset?q=&sort=metadata_modified+desc
```

```suggestion
    url: https://catalog.data.gov/dataset?q=&sort=metadata_created+desc
```
`Recently Added Datasets` vs `Recently Updated Datasets`